src/test/testRunner.ts

```diff
@@ -48,10 +48,18 @@ export function runTestsInFolder(testFolder: string): Promise<void> {
         const testFile = process.env['TEST_FILE'] === 'null' ? undefined : process.env['TEST_FILE']
         const testFilePath = testFile?.replace(/^src[\\\/]/, '')?.concat('.js')
 
-        const globalSetupPath = path.join(testsRoot, 'test', 'globalSetup.test.js')
-        if (testFilePath && fs.existsSync(globalSetupPath)) {
-            // XXX: explicitly add globalSetup, other tests depend on it.
-            mocha.addFile(globalSetupPath)
+        if (!integ) {
+            const globalSetupPath = path.join(testsRoot, 'test/globalSetup.test.js')
+            if (testFilePath && fs.existsSync(globalSetupPath)) {
+                // XXX: explicitly add globalSetup, other tests depend on it.
```

Comment:
XXX comments? what does that mean? remove?

Reply:
https://softwareengineering.stackexchange.com/a/65470/45819 . FIXME is not always appropriate, often something just needs to be called out as fragile/ugly but there is no plan to fix it.